### ISSUE TITLE
Share to Teams Button Accessibility Issue and Warning

### ISSUE THREAD

**Issue by paulconlin** [MicrosoftDocs/msteams-docs#763]
The Share to Teams button always throws a warning if there isn't an element readily on the page 
![image](https://user-images.githubusercontent.com/10995964/60472461-9690a000-9c2e-11e9-86d3-046acbd9b97a.png)

What do you recommend for use cases where the share button will not be rendered initially - ie in a modal? Can you suppress the warning in any way? I intend to just use the shareToMicrosoftTeams.renderButtons() when the modal opens.

**Comment by paulconlin** [comment]
Also noticing that the screen reader is not announcing properly on the element. It's reading the image tag because there's no alt tag or aria-hidden on that element. Do you have any suggestions on how to support an accessible experience? 

![image](https://user-images.githubusercontent.com/10995964/60472805-ee7bd680-9c2f-11e9-8dd4-96b88702fa1d.png)

**Comment by clearab** [comment]
For the first, there is no way to suppress the warnings, however using the renderButtons function is the correct approach.

For the second, that has been filed as a bug and will be fixed in a future release.

**Comment by paulconlin** [comment]
Thanks for your response, Andrew. I'm a little curious to know the intent of the warnings if the renderButtons function is also a correct way of utilizing the plugin?

Also good to know about item 2, I'll keep an eye out for that. Any chance it would be resolved before August 1st? That's when we plan to go live with it.

**Comment by drewatk** [comment]
Hi @paulconlin,
The warnings are simply intended for those who are using the basic feature, as the script scans the page on load. We'll consider making those warnings a bit quieter in a future release.

On the second point, it is a small fix for screen readers, so a fix before August is likely. We'll resolve this issue when it has been fixed.

Thank you for the feedback!

**Comment by clearab** [comment]
This should be fixed now, please let us know if you are still experiencing any issues.
